tests/js/lib/sequencer.js

```diff
@@ -0,0 +1,20 @@
+// Helper function to assemble usable test cases that pause at all the right times instead of either not running at all
+// or failing because they ran too early.
+//
+"use strict";
+var fluid = fluid || require("infusion");
+var gpii  = fluid.registerNamespace("gpii");
+fluid.registerNamespace("gpii.schema.tests");
+gpii.schema.tests.addRequiredSequences = function (sequenceStart, rawTests) {
```

Comment:
Since `gpii.express` is already updated, and since I only need this for tests that somehow involve express, I was thinking of putting it there.